packages/schemix/src/schemix/query/select.py

```
@@ -0,0 +1,236 @@
+from schemix.exceptions import QueryError
+from schemix.query import SQLExpression
+from schemix.table import BaseTable
+
+if TYPE_CHECKING:
+    from schemix.database import Database
+
+
+CType = TypeVar("CType", bound=Mapping[str, ColumnType | SQLExpression])
+
+
+class SelectBuilder[CType]:
```

Pylint (3.3.7)

[error] Parsing failed: 'invalid syntax (schemix.query.select, line 20)'

(E0001)